DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -64,7 +65,7 @@ public string FinancialYear
         public void Calculate()
         {
             CalculateBtn.Click();
-            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => InterestEarnedFld.GetAttribute("value") != null);
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => CalculateBtn.GetAttribute("disable") != string.Empty);
```

Comment:
Your solution is ok. Here are few other possible ways:
```suggestion
            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => CalculateBtn.Enabled);
            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(ExpectedConditions.ElementIsClickable(By.XPath("//button[@id='calculateBtn']")))
```